index.js

```diff
@@ -56,12 +56,16 @@ async function runLinterFromPRData (pullRequests, context, headSha) {
   const checkRunResponse = apiHelper.inProgressAPIresponse(context, headSha)
 
   try {
+    // For now only deal with one PR
+    const PR = pullRequests[0]
+
+    // Loads the Precaution configuration file
+    const rawObj = await apiHelper.getConfigFile(context, cache.getBranchPath(repoID, PR.id))
+    const configObj = rawObj ? JSON.parse(rawObj.data) : []
```

Comment:
Here we're assigning a JavaScript object `{}` if rawObj is non-null or an array `[]` otherwise. We should be consistent and assign the same type to this variable in both instances.

Reply:
With this, we are checking if the rawObj is null, undefined or blank:
https://stackoverflow.com/questions/5515310/is-there-a-standard-function-to-check-for-null-undefined-or-blank-variables-in

Reply:
Based on the discussion above on `undefined` vs. `[]` I agree that `undefined` (or `null`?) seems to make sense here

Reply:
I read through some opioons what is `null` and `undefined`:
![image](https://user-images.githubusercontent.com/16246778/56963152-f7cdc180-6b60-11e9-917c-e705a011dda8.png)
https://stackoverflow.com/questions/5076944/what-is-the-difference-between-null-and-undefined-in-javascript

and I think in this situation null is more appropriate because we are initializing the config obect with value of null which not `undefined` anymore because it's defined as null. :D